bootstrap-into-dart.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title: Bootstrap into Dart
+permalink: /bootstrap-into-dart/
+---
+
+New to the [Dart](https://www.dartlang.org) language?
+We've cherry-picked our favorite resources to
+help you quickly learn Dart.
+We [looked at a lot of languages](/faq/#why-did-flutter-choose-to-use-dart),
+and we found Dart easy and fun to learn, and we hope these resources make it
+easy for you to learn, too.
+
```

Comment:
True, but not sure how much it's maintained. So I didn't want to link to it here. I can trust more that the links I used in this page will be kept up to date.